usmqe_tests/api/tendrl/conftest.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf8 -*-
+
+import pytest
+
+from usmqe.api.tendrlapi.common import TendrlApi
+from usmqe.api.tendrlapi.common import TendrlAuth
+
+
+@pytest.fixture(scope="session")
+def default_session_credentials(request):
+    """
+    During setup phase, login default usmqe user account (username and password
+    comes from usm.ini config file) and return requests auth object.
+    Then during teardown logout the user to close the session.
+    """
+    api = TendrlApi()
+    auth = api.login(
+        pytest.config.getini("usm_username"),
+        pytest.config.getini("usm_password"))
+    yield auth
+    api.logout(auth=auth)
+
+
+@pytest.fixture(scope="session")
+def invalid_session_credentials(request):
```

Comment:
This fixture should be parametrized so there can be tested user with valid username and invalid token, vice versa and different roles (after roles will be implemented).